gson/src/main/java/com/google/gson/stream/JsonReader.java

```diff
@@ -1485,6 +1485,16 @@ public String getPath() {
     return result.toString();
   }
 
+  /**
+   * Return she current position of the input string
+   *
+   * @return Position of the json reader
```

Comment:
```suggestion
   * Return the current position of the JSON reader.
   *
   * @return Position of the JSON reader
```

Though this documentation should be improved because it appears the return value is actually the number of chars from the beginning of the current line (if I see that correctly?).